core.go

```diff
@@ -43,31 +70,43 @@ func (c *core) Check(ent zapcore.Entry, ce *zapcore.CheckedEntry) *zapcore.Check
 func (c *core) Write(ent zapcore.Entry, fs []zapcore.Field) error {
 	clone := c.with(fs)
 
-	event := sentry.NewEvent()
-	event.Message = ent.Message
-	event.Timestamp = ent.Time
-	event.Level = sentrySeverity(ent.Level)
-	event.Platform = "Golang"
-	event.Extra = clone.fields
-	event.Tags = c.cfg.Tags
-
-	if !c.cfg.DisableStacktrace {
-		trace := sentry.NewStacktrace()
-		if trace != nil {
-			trace.Frames = filterFrames(trace.Frames)
-			event.Exception = []sentry.Exception{{
-				Type:       ent.Message,
-				Value:      ent.Caller.TrimmedPath(),
-				Stacktrace: trace,
-			}}
+	/*
+		only when we have local sentryScope to avoid collecting all breadcrumbs ever in a global scope
+	 */
+	if c.cfg.EnableBreadcrumbs && c.cfg.BreadcrumbLevel.Enabled(ent.Level) && c.sentryScope != nil {
+		breadcrumb := sentry.Breadcrumb{
+			Data:      clone.fields,
+			Level:     sentrySeverity(ent.Level),
+			Message:   ent.Message,
+			Timestamp: ent.Time,
+			Type:      "default",
```

Comment:
I think we can drop this to make the payload smaller, as this is the default behavior.